### PR TITLE
[JENKINS-38987 Follow-up] Provide implementations of getPronoun() for the SCMHead/Source/Navigator

### PR DESCRIPTION
**Pull request by stephenc**
@reviewbybees

### CODE REVIEW

**Comment by abayer** [comment]
:bee:

File: pom.xml [hunk 1]
```diff
@@ -56,7 +56,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>credentials</artifactId>
-            <version>1.22</version>
+            <version>2.1.6</version>
```
Comment (jglick) [inline pom.xml]: Why?

File: src/main/java/org/jenkinsci/plugins/github_branch_source/BranchSCMHead.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package org.jenkinsci.plugins.github_branch_source;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import jenkins.scm.api.SCMHead;
+
+/**
+ * Head corresponding to a branch.
+ * @since FIXME
```
Comment (jglick) [inline src/main/java/org/jenkinsci/plugins/github_branch_source/BranchSCMHead.java]: Just make it package access.

File: pom.xml (hunk 1 above)
Comment (stephenc) [inline pom.xml]: Well needs to be at least 2.1.0 based on transitive dependencies, and probably should be 2.1.5 for big fixes... and 2.1.6 is zero risk fix on 2.1.5... so hence 2.1.6

File: src/main/java/org/jenkinsci/plugins/github_branch_source/BranchSCMHead.java (hunk 2 above)
Comment (stephenc) [inline src/main/java/org/jenkinsci/plugins/github_branch_source/BranchSCMHead.java]: I want this elsewhere

**Pull request merged**